### DISCUSSION TITLE
Lack of Italian VAT as an identifier for the electronic address system

### DISCUSSION THREAD

**Discussion by X0RN0T** [stephanstapel/ZUGFeRD-csharp#397] [General]
Why is Italian VAT missing as an identifier for the electronic address system? It is possible to select the VAT of almost all European countries, including the Vatican, but not that of Italy? Are there any regulations that I am not aware of or is it just missing?

**Comment by stephanstapel** [comment]
Hi @X0RN0T ,
If you are missing anything, I am happy to review your pull request.

**Reply by X0RN0T** [reply]
So it's not missing on purpose?

**Reply by stephanstapel** [reply]
Not from my side. Which list are you referring to? Maybe it is missing in the official code list?

**Reply by X0RN0T** [reply]
You are right, my mistake, it is missing in the official EAS code list - version 13.0

**Reply by stephanstapel** [reply]
oh my. That's really interesting. Thanks for finding that out. What will you do now?

**Reply by X0RN0T** [reply]
I guess nothing. The official code list says no, I will advise my user to use electronic mail instead. The Austrian VAT number is also missing. This would even be in the code list. It`s also possible that there is no “Italy VAT number” because in Italy it could be called “Generale imposta sul valore aggiunto”, for example, but that is beyond my knowledge.
